Add Travis CI configuration.

AlekSi opened the pull request.

AlekSi committed:
Add Travis CI configuration.

rumyantseva commented on Makefile:
```diff
@@ -9,7 +10,8 @@ watch:
 init:
 	brew update
-	brew install hugo
+	brew install hugo pre-commit
```
What if someone doesn't want to install `pre-commit` tool?

AlekSi replied:
Then that someone can run those commands by hands, nothing prevents they from doing that. 😄

rumyantseva replied:
Ok

AlekSi committed:
Update extensions list, log less, stop on error.

AlekSi committed:
Remove attribution.

AlekSi commented:
PTAL

AlekSi committed:
Also install go.

AlekSi committed:
Bump macOS version.

rumyantseva reviewed (approved):
LGTM

AlekSi committed:
Do not install Go

It is already installed on Travis CI, and we assume all contributors to the website has it installed already.
